pages/common/ed.md

```diff
@@ -28,7 +28,7 @@
 
 - Write the current document to a new file (the filename can be omitted if `ed` was called with an existing file):
 
-`w {{filename.txt}}`
+`w {{file}}`
```

Comment:
I'd rather keep "filename", which IMO makes it slightly clearer that this doesn't accept a path.